libqtile/widget/volume.py

```diff
@@ -147,14 +154,18 @@ def _update_drawer(self):
             self.drawer.ctx.set_source(self.surfaces[img_name])
             self.drawer.ctx.paint()
         elif self.emoji:
+            if len(self.emoji_list) < 4:
+                self.emoji_list = ["\U0001f507", "\U0001f508", "\U0001f509", "\U0001f50a"]
+                logger.warning("Emoji list given has less than 4 items. Falling back to default emojis.")
```

Comment:
I'm not 100% certain what the CI was complaining about, but if you run `black` on your code it will fix it for you. My guess is the line is too long:

```suggestion
                logger.warning(
                    "Emoji list given has less than 4 items. Falling back to default emojis."
                )
```